content/applications/marketing/events/event_booths.rst

```diff
@@ -0,0 +1,242 @@
+
+On the :guilabel:`Booth Category` page, the following information for each booth category is listed:
+
+- :guilabel:`Name`: the name of the booth category.
+- :guilabel:`Create Sponsor`: if the box in this column is ticked, that means when a booth with this
+  category is booked, a sponsor is created for the user who booked it.
+- :guilabel:`Product`: the *Event Booth* product associated with that specific booth category.
+- :guilabel:`Price`: the price of a booth in that booth category.
+
+When the :guilabel:`additional options` :icon:`oi-settings-adjust` icon, located to the far-right of
+the column titles, is clicked, a drop-down menu of additional column options appears. From the
+resulting drop-down menu, tick the box beside :guilabel:`Sponsor Level` and/or :guilabel:`Sponsor
+Type` to reveal those columns on the :guilabel:`Booth Category` page.
+
+To edit an existing booth category, select it from the list, and proceed to make any desired
+modifications from the event category form.
+
+Create booth category
+---------------------
+
+To create a booth category from the :guilabel:`Booth Category` page, click the :guilabel:`New`
+button in the upper-left corner to reveal a blank booth category form.
+
+.. image:: event_booths/booth-category-form.png
+   :align: center
+   :alt: A typical booth category form in the Odoo Events application.
+
+Start by entering a name for the booth category in the top :guilabel:`Booth Category` field. This is
+a **requried** field.
+
+To add a corresponding image to the booth category (e.g. a sample photo of how the booth looks),
+click the :icon:`fa-pencil` :guilabel:`(pencil)` icon that appears when the cursor hovers over the
+camera placeholder in the upper-right corner of the booth category form. When clicked, proceed to
+upload the desired image to the booth category form, if needed.
+
+In the :guilabel:`Booth Details` section, users **must** assign a :guilabel:`Product` to the
+category, and it **must** have *Event Booth* set as the *Product Type* on the product form.
+
+And, regardless of the listed price on the *Event Booth* product chosen, the user can input a custom
+:guilabel:`Price` to be applied for this booth category in the field below.
+
+In the :guilabel:`Sponsorship` section, there is a :guilabel:`Create Sponsor` checkbox option. When
+that checkbox is ticked, that means that when a booth belonging to this booth category is booked,
+the user is created as an official *Sponsor* of the event.
+
+Also, when the :guilabel:`Sponsorship` checkbox is ticked, two additional fields appear beneath it:
+:guilabel:`Sponsor Level` and :guilabel:`Sponsor Type`.
+
+Select a desired level of sponsorship from the :guilabel:`Sponsor Level` drop-down field.
+
+.. tip::
+   To modify any existing :guilabel:`Sponsor Level`, select it from the drop-down field, then click
+   the :guilabel:`➡️ (right arrow)` that appears at the end of the line. Doing so a separate page,
```

Comment:
Is there an fa-icon that this guilabel emoji can be replaced with? So that we're consistently using either odoo icons or fa-icons throughout the doc